quantum/quantum_keycodes.h

```diff
@@ -716,9 +716,11 @@ enum quantum_keycodes {
 #define MACROTAP(kc) (QK_MACRO | (FUNC_TAP << 8) | (kc))
 #define MACRODOWN(...) (record->event.pressed ? MACRO(__VA_ARGS__) : MACRO_NONE)
 
-#define KC_GESC GRAVE_ESC
+#define QK_GESC QK_GRAVE_ESCAPE
 
-#define EEP_RST EEPROM_RESET
+#define QK_BOOT QK_BOOTLOADER
+#define QK_DBUG QK_DEBUG_TOGGLE
+#define QK_CLEE QK_CLEAR_EEPROM
```

Comment:
I'm not a fan of `QK_CLEE`. It's name isn't exactly clear. and just sounds weird.

Reply:
Yeah... I've been trying to keep the short aliases as much as possible a "condensed" version of the long form keycodes, but I'm open to suggestions.